phylogenetic/config/description.md

```diff
@@ -18,7 +18,7 @@ Our bioinformatic processing workflow can be found at [github.com/nextstrain/mon
 
 #### Underlying data
 We curate sequence data and metadata from the [NCBI Datasets command line tools](https://www.ncbi.nlm.nih.gov/datasets/docs/v2/download-and-install/),
-using NCBI Taxonomy ID "10244", as starting point for these analyses.
+using an NCBI Taxonomy ID defined in [ingest/config/config.yaml](https://github.com/nextstrain/monkeypox/blob/master/ingest/config/config.yaml), as starting point for these analyses.
```

Comment:
I realized I could have submitted this as a separate PR. But would this wording be okay? @joverlee521 ?

Reply:
LGTM!